tracing-example/pom.xml

```diff
@@ -52,31 +53,8 @@
             <id>wildfly27</id>
             <properties>
                 <version.org.wildfly>27.0.1.Final</version.org.wildfly>
+                <dep.tracing.scope>compile</dep.tracing.scope>
```

Comment:
@jamezp I have to use `compile` scope instead of `runtime` for wildfly27 or the project won't compile.